pkg/systemd/init.js

```diff
@@ -397,17 +397,19 @@ $(function() {
         $("#services-text-filter").on("input", render);
         $(document).on("click", "#clear-all-filters", clear_filters);
 
+        const relevant_suffices = new Set(["service", "target", "socket", "timer", "path"]);
```

Comment:
It might be worthwhile to avoid this explicit list here.  We can put it into services.jsx at least.

Reply:
Done.